hooks/pre-commit.d/cargo-audit

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+if [[ ! -z $TRAVIS && ! $1 ]]; then
```

Comment:
```suggestion
[[ -z "$TRAVIS" || "$1" == "--force" ]] && cargo audit
```

Reply:
Done. _Someday_ I'll make an elegant shell script on the first try...

Reply:
@npmccallum Hm... There is one case where both conditions are "false" (where it's running on Travis, so `TRAVIS` exists, and `--force` is not enabled) and I think that's causing a failure.

I've amended the statement to look for the one case we want to exclude. It's similar to what I had before, but certainly less verbose -- and it works again.